src/compas/geometry/_intersections/intersections.py

```diff
@@ -383,6 +384,40 @@ def intersection_segment_plane(segment, plane, tol=1e-6):
     return None
 
 
+def intersection_polyline_plane(polyline, plane, expected_number_of_intersections=None, tol=1e-6):
+    """Calculate the intersection point of a plane with a polyline. Reduce expected_number_of_intersections to speed up.
+
+    Parameters
+    ----------
+    polyline : compas.geometry.Polyline
+        polyline to test intersection
+    plane : compas.Geometry.Plane
+        plane to compute intersection
+    tol : float, optional
+        A tolerance for membership verification.
+        Default is ``1e-6``.
+
+    Returns
+    -------
+    list of :class:`compas.geometry.Point`
```

Comment:
this is not correct.
functions return native Python objects (not COMPAS objects)
```
list of points
```